src/extensions/react-flow/edges/EdgeDefault.tsx

```diff
@@ -11,6 +11,7 @@ import { drawEdgeStraight} from "./utils";
 
 export interface EdgeDefaultDataProps {
     pathGlowWidth?: number;
+    inversePath?: boolean;
```

Comment:
It does not inverse the path, it just switches the marker to get used on the opposite path ending. The property name should reflect this, and it may also have upcoming changes in mind.